src/lesson2_methodsoverloading/QualificationManager.java

```diff
@@ -0,0 +1,50 @@
+package lesson2_methodsoverloading;
+
+public class QualificationManager {
+    public static String getQualification(double score) {
+        if (score < 50) {
+            return "Failed the test";
+        } else if (score > 50 & score < 60) {
+            return "Good!";
+        } else if (score > 60 & score < 70) {
+            return "Good job";
+        } else if (score > 70 & score < 80) {
+            return "You are on fair";
+        } else if (score > 80 & score < 90) {
+            return "Very good";
+        } else if (score > 90 & score <= 100) {
+            return "Exceleny";
+        } else if (score < 1 & score > 101) {
+            return "This score is not valid.";
+        }
+        return null;
```

Comment:
You could return "Invalid input" instead of null. I guess in this way the output is more suggestive